pandas/tests/series/test_constructors.py

```diff
@@ -574,6 +574,12 @@ def test_constructor_with_datetime_tz(self):
         expected = Series(pd.DatetimeIndex(['NaT', 'NaT'], tz='US/Eastern'))
         assert_series_equal(s, expected)
 
+        # GH 17415: With naive string
+        ts_naive = '2013-01-01 00:00:00'
+        result = Series([ts_naive], dtype='datetime64[ns, CET]')
```

Comment:
make this a separate test

parametrize over adding a NaT, nan, None here as well

Reply:
```
In [3]: Series([pd.NaT], dtype='datetime64[ns, CET]') # same for np.nan and None
Out[3]: 
0   NaT
dtype: datetime64[ns, CET]

In [4]: Series(pd.Timestamp(pd.NaT, tz='CET')) # same for np.nan and None
Out[4]: 
0   NaT
dtype: datetime64[ns]
```

So I figure both of these should be equivalent and `In[4]` should be the correct behavior?